src/mscorlib/corefx/System/Globalization/TextInfo.cs

```diff
@@ -38,38 +36,28 @@ public partial class TextInfo : ICloneable, IDeserializationCallback
         ////
 
         [OptionalField(VersionAdded = 2)]
-        private String m_listSeparator;
```

Comment:
@stephentoub 
I don't think you are over estimating the cost. I know the scope is really big. my only point here is looks from what you said is we are not going to support binary serialization cross-plat, but it is still not clear to me if we are going to support DCS cross-plat. if decided we support DCS cross plat then we are going to do the work anyway (regardless of the  scope) and at that time it could make sense to support binary serialization too. 

anyway, I'll talk to DCR guys to know what they decided and if they have similar decision or they willing to support cross-plat. I'll CC you in my communications.